application/comit_node/src/logging.rs

```diff
@@ -14,19 +15,21 @@ pub fn set_context<S: ToString>(input: &S) {
     });
 }
 
-pub fn set_up_logging() {
+pub fn set_up_logging(settings: &ComitNodeSettings) {
     Dispatch::new()
         .format(move |out, message, record| formatter(out, message, record))
-        // TODO: get level from config file once implemented with #136
         .level(LevelFilter::Debug)
```

Comment:
:wrench: What about the other way round:

1. Make the standard level be controlled by a single flag in the config file.
2. Remove all those `level_for` specific to the application here.
3. Override certain libraries (like tokio) that we don't want.

Reply:
I wouldn't mind having a single default one. Would you remove all the section specific ones in favour of it or just make it set the default?

Reply:
Yes I would remove them so they are all automatically set to the default one.